Safety/Enter-Observatory-Spaces/Enter-AuxTel-Telescope.rst

```diff
@@ -0,0 +1,190 @@
+  
+
+.. _Safe-entry-to-AuxTel-access-during-ventilation:
+
+Access During Ventilation
+=========================
+
+
+.. figure:: ./_static/vent.png
+    :name: Venting
+
+    Air extraction fan.
+
+
+
+
+1. **Ventilation Notification**
+
+   - Ventilation starts with a notification via Slack channels and/or radio channel 3.
+
+
+
+2. **Safety Precautions**
+
+   - Ensure the windows are fully open and the fan operates at 20%.
+
+   - Take special care not to lean on the windows or manipulate the fan while it is running.
+
+   - On the telescope's second floor, verify that the telescope is in a horizontal position and the dome is facing opposite the Sun to maximize the ventilation system.
+
+
+
+.. _Safe-entry-to-AuxTel-access-during-calibrations:
+
+Access During Calibrations
+==========================
+
+.. warning::
+
+    Access is **NOT** permitted during calibrations unless there is an issue with the system that requires attention, and images can be interrupted or are not currently being taken. 
+    Ensure there are no announcements in Slack, and confirm with summit staff (daytime shift workers or an Observing Specialist) that no images are in progress.
+
+
+
+.. _Safe-entry-to-AuxTel-access-during-night-time-operations:
+
+Access During Night time Operations
+===================================
+
+1. **Entry Permissions**
+
+   - Only authorized personnel and the Observing Specialist team can enter without an external permit during operations.
+
+
+2. **Lighting Restrictions**
+   
+   .. warning:: 
+
+    During operations, it is prohibited to turn on building lights when entering the telescope.
+
+
+   .. caution:: 
+        
+    The use of flashlights or headlamp is permitted but they should point towards the ground and should be minimized to avoid interfering with observations.
+
+
+   .. tip:: 
+
+    Red light flashlights can be a good choice and they interfere much less than regular white light.
+
+
+.. _Safe-entry-to-AuxTel-behaviour-standards:
+
+Behavior Standards
+==================
+
+Maintain proper behavior at all times, including:
+
+- Being close to a radio (or always carrying one) and communicating your intentions to summit staff and on Slack.
+  
+- Following the `safety gate procedure <https://obs-ops.lsst.io/v/SITCOM-1360/Safety/Safety-Systems/AuxTel-Safety-Gate-Introduction.html>`__.
+  
+- Locking out movement when necessary.
+
+
+.. _Safe-entry-to-AuxTel-Emergency-Procedures:
+
+Emergency Procedures
+====================
+
+In case of any emergency, notify immediately via **radio channel 3 for Rubin staff, or channel 1 for after hours emergencies.**
+For detailed instructions on specific types of emergencies, please refer to the :ref:`Emergency Response Guide <Safety-emergency-response-guide>`.
+Remember to always keep the list of :ref:`Emergency Numbers <summit-contact-numbers-emergency-numbers>` handy for any urgent situations.
+
+
+This procedure was last modified on |today|.
```

Comment:
It's a great document; it is clear, readable, concise, and well-detailed. I only have one comment to review on line 90 ( a suggestion). 

Thank you very much for this document.